### DISCUSSION TITLE
Labs: naked hero 8 broken BEC, so i cant hit the record button anymore

### DISCUSSION THREAD

**Discussion by maxcineflights** [gopro/labs#226] [Q&A]
Naked hero 8 broken BEC, so i cant hit the record button anymore (i am in thailand traveling and dont want to wait 2 weeks for a new bec + 35 dollars shipping)

I use all the time now the gopro app but the connectiong is a pure nightmare often i have to try 5 times its so terrible...

I know there  is the Labs: IMU detect...but i dont get the settings right: start: 1 (start really smoothly and want to have a reliable recording) + start delay 0 seconds .... that works...but i dont really understand the "end sensitivity" (the reccomended options are Start:4, and end sensitivity of 8) but i can not understand the End sens, why can i not only make a checkbox: stop recording when no new gyrodata is there?? I really want to have a reliable start and stop because i do payed indoor flying tours in Thai villas right now... 
and the "hold function" is also a little confusing to me.... i think hold is if no new gyrodata is coming then it holds the reocrding for example 5 secoonds but how does that relate then to the end "sensitivity"

I want to start, fly and lans pretty smooth with my sub25g and still need it reliable...is there any way= dont get it right with my testing...

or is there maybe a possible way to start and stopp automatically as soon as i power on? (then there still is the problem with the corrupted files if i cant stop it withouth app...

or is there a possible way to use a qr to start recording and a qr to stopp it somehow?

Thank you so much...(use a older Labs fimware, have seen the hero8 also got an update with the labs) = i havent dont it jet, because i really need to be able to film and i am a little scared that the update makes things maybe worse for me with the app conneciton or anything, because now i still can use it at least even with a lot of headaches...
Thank you so much!!

**Comment by dnewman-gpsw** [comment]
If you have any GoPro Labs camera with an LCD, much of this is self-explanatory, as the LCD reports the IMU numbers, so you can see how all these thresholds work. I use IMU triggers for my GoPro Bones FPV rigs, for my Naked HERO11 which still as a microphone I use audio triggers (drones are noisy.) For an IMU trigger, I use a low sensitivity to start, low meaning small change will not trigger, and I punch the throttle on take-off to ensure a capture trigger  (0.125g threshold 4.) Once flying smooth, the IMU will fall in magnitude, as I don't want the capture to stop, so I set the end threshold to 8 (0.008g,) so just the normal turbulence and motor vibrations keep the recording going. Hold sets how long to continue recording after landing, also can be help for some short moments of low IMU activity, like some unpowered dives.        

Latest Labs firmware for HERO8 does support !MWAKE=2 , auto boot on power, then combined with a BOOT command, you can have the camera auto-start the IMU trigger.  I will never need to press a button, and always get the shoot.  

https://gopro.github.io/labs/control/set/?cmd=!MQRDR=1!MBOOT="!Lboot"!SAVEboot=mV!SI4-8D1H5!R

**Reply by maxcineflights** [reply]
thank you very much. 
I have the a naked hero 8, but its a little too unreliable for me, tested different things over and over again.
I now go simply with the start and end command over a QR Code, because its for Client Projects. Thank you so much for your Input.
